ports/stm32/boards/stm32h743_af.csv

```diff
@@ -29,7 +29,7 @@ PortB,PB9,,TIM17_CH1,TIM4_CH4,DFSDM_DATIN7,I2C1_SDA,SPI2_NSS/I2S2_WS,I2C4_SDA,SD
 PortB,PB10,,TIM2_CH3,HRTIM_SCOUT,LPTIM2_IN1,I2C2_SCL,SPI2_SCK/I2S2_CK,DFSDM_DATIN7,USART3_TX,,QUADSPI_BK1_NCS,OTG_HS_ULPI_D3,ETH_MII_RX_ER,,,LCD_G4,EVENTOUT,
 PortB,PB11,,TIM2_CH4,HRTIM_SCIN,LPTIM2_ETR,I2C2_SDA,,DFSDM_CKIN7,USART3_RX,,,OTG_HS_ULPI_D4,ETH_MII_TX_EN/ETH_RMII_TX_EN,,,LCD_G5,EVENTOUT,
 PortB,PB12,,TIM1_BKIN,,,I2C2_SMBA,SPI2_NSS/I2S2_WS,DFSDM_DATIN1,USART3_CK,,CAN2_RX,OTG_HS_ULPI_D5,ETH_MII_TXD0/ETH_RMII_TXD0,OTG_HS_ID,TIM1_BKIN_COMP12,UART5_RX,EVENTOUT,
-PortB,PB13,,TIM1_CH1N,,LPTIM2_OUT,,SPI2_SCK/I2S2_CK,DFSDM_CKIN1,USART3_CTS_NSS,,CAN2_TX,OTG_HS_ULPI_D6,ETH_MII_TXD1/ETH_RMII_TXD1,,,UART5_TX,EVENTOUT,
+PortB,PB13,,TIM1_CH1N,,LPTIM2_OUT,,SPI2_SCK/I2S2_CK,DFSDM_CKIN1,USART3_CTS,,CAN2_TX,OTG_HS_ULPI_D6,ETH_MII_TXD1/ETH_RMII_TXD1,,,UART5_TX,EVENTOUT,
```

Comment:
There are lots of these... I think it's correct as it was and this indicates that the USART supports a slave select pin in synchronous slave mode.  The proper way to fix it would be for boards/make-pins.py to understand this.

Reply:
The way that I've been dealing with this is to change it from UART3_CTS_NSS to be UART3_CTS/UART3_NSS

Then makepins,py works for detecting the CTS behaviour and when/if NSS support is added then that will work too.

There are many cases of multiple functions on a single pin that I think makepins.py would have difficulty parsing using the pin label from the datasheet.

Reply:
Ok, that sounds like a good way to proceed.  It looks like we already do this in some places, eg `TIM2_CH1_ETR` becomes `TIM2_CH1/TIM2_ETR` (for PA0 of F405).

Another case is `USART3_RTS_DE`